frontend/src/renderer/App.tsx

```diff
@@ -60,20 +61,24 @@ export default function Hello() {
 
   return (
     <div>
-      <MantineProvider>
-        <Router>
-          <HeaderResponsive links={randomLinks} />
-          <Routes>
-            <Route path="/" element={isConfigValid ? <GettingStarted /> : <Config />} />
-            <Route path="/upload" element={<Upload />} />
-            <Route path="/chat" element={<Chat />} />
-            <Route path="/config" element={<Config />} />
-            <Route path="/summary" element={<Summary />} />
-
-          </Routes>
-          <Footer />
-        </Router>
-      </MantineProvider>
+      <Provider store={store}>
+        <MantineProvider>
+          <Router>
+            <HeaderResponsive links={randomLinks} />
+            <Routes>
+              <Route
+                path="/"
+                element={isConfigValid ? <GettingStarted /> : <Config />}
+              />
+              <Route path="/upload" element={<Upload />} />
+              <Route path="/chat" element={<Chat />} />
+              <Route path="/config" element={<Config />} />
+              <Route path="/summary" element={<Summary />} />
+            </Routes>
+            <Footer />
+          </Router>
+        </MantineProvider>
+      </Provider>
```

Comment:
Why did you change something here?

Reply:
i didn't change anything, i added <Provider> to make the state accessible for the whole app (redux)